cli/src/generate/build_tables/build_parse_table.rs

```diff
@@ -972,13 +988,19 @@ pub(crate) fn build_parse_table<'a>(
         inlines,
         &item_set_builder,
     );
+    let actual_conflicts = syntax_grammar
+        .expected_conflicts
+        .clone()
+        .into_iter()
+        .collect();
```

Comment:
For this place I'd suggest to rewrite it like in the commit: https://github.com/amaanq/tree-sitter/commit/9aa0b08ccd073782256dd0d2948edf0dad21c464